src/Umbraco.Core/Umbraco.Core.csproj

```diff
@@ -55,7 +55,11 @@
   </ItemGroup>
   <ItemGroup>
     <!-- note: NuGet deals with transitive references now -->
-    <PackageReference Include="AutoMapper" Version="8.0.0" />
+    <PackageReference Include="Umbraco.Code">
```

Comment:
Maybe add a note/comment that this is a Roslyn Analyzer NuGet package & what its use is for please. So the entire team knows what this is & how its used.

Reply:
Bump - think be good to describe this & your magic comment thing @zpqrtbnk